token/services/db/sql/identity.go

```diff
@@ -197,41 +242,47 @@ func (db *IdentityDB) signerInfoExists(h string) (bool, error) {
 	return true, nil
 }
 
-type WalletPathStorageIterator struct {
+type IdentityConfigurationIterator struct {
 	rows              *sql.Rows
 	configurationType string
 }
 
-func (w *WalletPathStorageIterator) Close() error {
+func (w *IdentityConfigurationIterator) Close() error {
 	return w.rows.Close()
 }
 
-func (w *WalletPathStorageIterator) HasNext() bool {
+func (w *IdentityConfigurationIterator) HasNext() bool {
 	return w.rows.Next()
 }
 
-func (w *WalletPathStorageIterator) Next() (driver.IdentityConfiguration, error) {
+func (w *IdentityConfigurationIterator) Next() (driver.IdentityConfiguration, error) {
 	var c driver.IdentityConfiguration
 	c.Type = w.configurationType
-	err := w.rows.Scan(&c.ID, &c.URL)
+	err := w.rows.Scan(&c.ID, &c.URL, &c.Config, &c.Raw)
 	return c, err
 }
 
 func (db *IdentityDB) GetSchema() string {
 	return fmt.Sprintf(`
 		-- IdentityConfigurations
 		CREATE TABLE IF NOT EXISTS %s (
-			id TEXT NOT NULL PRIMARY KEY,
+			id TEXT NOT NULL,
             type TEXT NOT NULL,  
-			url TEXT NOT NULL
+			url TEXT NOT NULL,
+			conf BYTEA,
+			raw BYTEA,
+			PRIMARY KEY(id, type)
 		);
 		CREATE INDEX IF NOT EXISTS idx_ic_type_%s ON %s ( type );
+		CREATE INDEX IF NOT EXISTS idx_ic_id_type_%s ON %s ( id, type );
 
-		-- AuditInfo
+		-- IdentityInfo
 		CREATE TABLE IF NOT EXISTS %s (
             identity_hash TEXT NOT NULL PRIMARY KEY,
 			identity BYTEA NOT NULL,
-			info BYTEA NOT NULL
+			identityAuditInfo BYTEA NOT NULL,
```

Comment:
I'd use snake case for columns. they are case insensitive

Reply:
👍